packages/launchpad/src/global/GlobalEmpty.vue

```diff
@@ -2,28 +2,55 @@
   <main class="text-center">
     <h1 class="text-2rem mb-2">{{ t('globalPage.empty.title') }}</h1>
     <p class="text-lg font-light text-gray-600 mb-6">{{ t('globalPage.empty.helper') }}</p>
-    <button
-      type="button"
-      @click="selectProject"
-      class="min-w-220px relative block w-full border-2 bg-gray-50 border-gray-300 border-dashed rounded-lg p-12 text-center hover:border-gray-400 text-center"
-    >
-      <IconPlaceholder
-        class="-mb-8px mx-auto max-w-65px h-full relative justify-center w-full text-indigo-600"
-      />
-      <span class="mt-13px block text-lg font-medium text-gray-700 font-light">
-        <i18n-t keypath="globalPage.empty.dropText">
-          <a class="font-normal" @click="selectProject">{{ t('globalPage.empty.browseManually') }}</a>
-        </i18n-t>
-      </span>
-    </button>
+    <file-selector v-model="files" v-slot="{ openDialog }" allow-multiple>
+      <dropzone v-slot="{ hovered }" @click="openDialog">
+        <div
+          class="min-w-220px relative block w-full border-2 bg-gray-50 border-gray-300 border-dashed rounded-lg p-12 text-center hover:border-gray-400 text-center"
+          :class="{ 'border-blue-200': hovered }"
+        >
+          <IconPlaceholder
+            class="mx-auto max-w-65px h-full relative justify-center w-full text-indigo-600"
+          />
+          <i18n-t keypath="globalPage.empty.dropText">
+            <button
+              class="text-indigo-600 hover:underline"
+            >
+            <!-- 
+              This button allows keyboard users to fire a click event with the Enter or Space keys, 
+              which will be handled by the dropzone's existing click handler.
+             -->
+            {{ t('globalPage.empty.browseManually') }}</button>
+          </i18n-t>
+        </div>
+      </dropzone>
+    </file-selector>
+    <div data-testid="upload-name" class="hidden">{{uploadName}}</div>
   </main>
 </template>
 
 <script lang="ts" setup>
 import { useI18n } from "../composables"
 import IconPlaceholder from 'virtual:vite-icons/icons8/circle-thin'
+import { FileSelector, Dropzone } from 'vue3-file-selector'
+import { ref, watch, onMounted } from 'vue'
 
 const { t } = useI18n()
+const files = ref([])
+const uploadName = ref('')
+
+
+const selectProject = (file) => { uploadName.value = file.name}
```

Comment:
File type should be available, I think it's just `File`

```suggestion
const selectProject = (file: File) => { uploadName.value = file.name}
```